### PR TITLE
Hangout: fix usage of STUN function

### PR DESCRIPTION
**Pull request by IvanNardi**

### CODE REVIEW

**Comment by IvanNardi** [comment]
It is incredible that this error didn't trigger any errors so far!

**Comment by utoni** [comment]
> It is incredible that this error didn't trigger any errors so far!

Google Hangout is in it's transition phase to Google Chat. It is probably not much used in these days. But this is just dangerous half-knowledge. 🦄

File: src/lib/protocols/hangout.c [hunk 1]
```diff
@@ -84,12 +84,13 @@ static u_int8_t is_google_flow(struct ndpi_detection_module_struct *ndpi_struct,
 void ndpi_search_hangout(struct ndpi_detection_module_struct *ndpi_struct,
 			 struct ndpi_flow_struct *flow) {
   struct ndpi_packet_struct * packet = &flow->packet;
+  int matched_src = 0;
```
Comment (utoni) [inline src/lib/protocols/hangout.c]: Scope of `matched_src` can be reduced if I am not wrong. (reading source code from a smartphone is sometimes confusing)

Comment (IvanNardi) [reply src/lib/protocols/hangout.c]: Done in the last push. Thanks

**Pull request merged**